addons/api/package.json

```diff
@@ -49,6 +50,7 @@
     "ember-cli": "^4.12.1",
     "ember-cli-dependency-checker": "^3.3.1",
     "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-cli-mirage": "3.0.3",
```

Comment:
We have this now both in `devDependencies` and `dependencies`

Reply:
Apologies. I will fix that now.

Reply:
I'm confused, why do we now have one pinned to `3.0.3` in `devDependencies` and `^3.0.3` in `dependencies`?

Reply:
I did some git stashing and I missed the devDep one. Let me try this again. Apologies, I thought I had it all fixed.

Reply:
Fixed and test build ran successfully and local tests working.